unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/Transform/ECSTransformPlugin.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using DCL.ECSRuntime;
+
+namespace DCL.ECSComponents
+{
+    public class ECSTransformPlugin : IDisposable
```

Comment:
Shouldn't this be a `IPlugin`?

Reply:
this class will be instantiated inside the ecs7 plugin, it won't be added directly to the plugin factory